test/scan.test.ts

```diff
@@ -124,7 +124,7 @@ describe('scan', () => {
   it('should throw exception when invalid options', async () => {
     expect.assertions(1);
     const expected = new Error(
-      `Could not scan C/C++ project, invalid options no path provided.`,
+      `Could not scan C/C++ project, invalid options: no path provided.`,
```

Comment:
I would argue that it's nice to use a common pattern from unix (and C, Go), to keep messages concatenated with : between all errors (which is an error as well). In this case `error: reason: original error`

So in that case it would be `Could not scan C/C++ project: invalid options: no path provided.`

But's that's just my opinion and it's probably not the pattern we have since before :)

Reply:
Ah interesting - I was just correcting the punctuation :D but if we can follow an established pattern, even better!

Reply:
Let's follow Snyks policies, I don't know how this is handled since before... or how it's handled by aggregated loggings etc. Do you know if we have established policies related to this?

Reply:
I'm not aware of any such conventions, but I think that using the pattern you suggest above would be nice and clear for now. I can do a bit of digging about logging conventions and update if it turns out we follow them.